client/src/components/Shop/productList.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+// TODO: Create products db to hold prod data
+// FIXME: import API from "../../utils/API";
```

Comment:
Worthwhile discussion as to the merits of inline todos vs tickets/issues in the project backlog.